Update authentication-plugin.rst (#6352)
cakephp/docs: en/tutorials-and-examples/cms/authentication-plugin.rst

        $this->Authentication->addUnauthenticatedActions(['login']);
    }

>     public function login()

Reviewer: Shouldn't this brace stay?

Author: https://github.com/cakephp/docs/pull/6351

Reviewer: This diff is removing the brace again though.

Now you can visit ``/users/logout`` to log out. You should then be sent to the login
page.

Customizing Login / Logout
=========================

You can also customize your login and logout screen very easily in CakePHP. 

> Add 'logout' as in the example below:

Reviewer: ```suggestion
Add 'logout' as in the example below::
```

    // in src/Controller/AppController.php
    public function beforeFilter(\Cake\Event\EventInterface $event)
    {
        parent::beforeFilter($event);
        // for all controllers in our application, make index and view
        // actions public, skipping the authentication check

>         $this->Authentication->addUnauthenticatedActions(['index', 'view', 'logout');

Reviewer: ```suggestion
        $this->Authentication->addUnauthenticatedActions(['index', 'view', 'logout']);
```

>         $this->Authentication->addUnauthenticatedActions(['index', 'view', 'logout']);

Reviewer: Any reason you are going with this approach over allowing logout in the UsersController?

Author: I wanted a logout screen, so I did it (below) because I don't know how to do it differently. But you can teach me how to do it right.

   public function beforeFilter(\Cake\Event\EventInterface $event)
    {
        parent::beforeFilter($event);
        // configure the login action to don't require authentication, preventing
        // the infinite redirect loop issue
        $this->Authentication->addUnauthenticatedActions(['login']);
        $this->Authentication->addUnauthenticatedActions(['logout']);
    }

Author: How do I cancel this change?

    public function login() {
    public function login()

Author: you can remove on logout if you wish

Reviewer 2: you have to use latest version to work on it
or now you can re-base it , or add it again with new commit

Reviewer: You can add logout and login to the same addUnauthenticatedActions() call in the UsersController beforeFilter method.

    }

Add in the login action to the UsersController class::

    public function login()
    {
        // Set the layout
        $this->viewBuilder()->setLayout('default_login');
        ...
    }
  

> You can copy the file 'layout/default.php' to 'layout/default_login.php' and change it or you can create your own custom file.

Reviewer: ```suggestion
You can copy the file **layout/default.php** to **layout/default_login.php** and change it or you can create your own custom file.
```

Follow the same procedure to change the logout screen if desired.

> More informations:: https://book.cakephp.org/4/en/views.html#layouts

Reviewer: ```suggestion
See :ref:`view-layouts`
```

We can use a ref anchor as they won't break in the future without a warning.
